runtime/near-test-contracts/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "near-test-contracts"
+version = "0.0.0"
```

Comment:
Does 0.0.0 make sense?

Reply:
It does -- we are not going to publish this crate, so it doesn't have a meaningful semver. So putting a default `0.1.0` would be (a minor and insignificant) lie. `0.0.0` and `publish = false` explicitly say that we don't care about semver in this case.